opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -234,7 +235,11 @@ mod tests {
             .with_simple_exporter(exporter.clone())
             .build();
 
-        let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
+        let non_opentelemetry_filter = tracing_subscriber::filter::filter_fn(|metadata| {
+            !metadata.target().starts_with("opentelemetry")
+        }); // filter internal logs
+        let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider)
```

Comment:
nit: lets move this to a common helper, to avoid the repetition

Reply:
This is done now.